### PR TITLE
Rename updateConnection -> heartbeat

### LINKED ISSUE #370: conntrack: Rename "updateConnection" to "heartbeatConnection"
Currently, in the context of the conntrack module, the word "update" is overloaded.
On the one hand, it is used in the record type "updateConnection" to describe periodic reports of active connections.
On the other hand, it used to describe updating connections stats with incoming flow logs.

Following the suggestion of https://github.com/netobserv/flowlogs-pipeline/pull/357#discussion_r1087254079, we can rename the first case and use the word "heartbeat" instead.

### PR DESCRIPTION
**Pull request by ronensc**
Fixes #370 

This PR should be merge after #357

### CODE REVIEW

File: pkg/api/conntrack.go [hunk 1]
```diff
@@ -37,10 +37,10 @@ type ConnTrack struct {
 }
 
 type ConnTrackOutputRecordTypeEnum struct {
-	NewConnection    string `yaml:"newConnection" doc:"New connection"`
-	EndConnection    string `yaml:"endConnection" doc:"End connection"`
-	UpdateConnection string `yaml:"updateConnection" doc:"Update connection"`
-	FlowLog          string `yaml:"flowLog" doc:"Flow log"`
+	NewConnection string `yaml:"newConnection" doc:"New connection"`
+	EndConnection string `yaml:"endConnection" doc:"End connection"`
+	Heartbeat     string `yaml:"heartbeat" doc:"Heartbeat"`
```
Comment (ronensc) [inline pkg/api/conntrack.go:42]: Is `Heartbeat` too casual? Is `HeartbeatConnection` preferred?

**Review by KalmanMeth — APPROVED** [review]

File: pkg/api/conntrack.go (hunk 1 above)
Comment (jpinsonneau) [reply pkg/api/conntrack.go:42]: What about `TickConnection` ?

**Comment by jpinsonneau** [comment]
This will impact https://github.com/netobserv/network-observability-operator/pull/252 and https://github.com/netobserv/network-observability-console-plugin/pull/285

**Review by jpinsonneau — APPROVED** [review]
LGTM, thanks @ronensc !

**Pull request merged**